go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/graph-gophers/graphql-go v1.5.1-0.20230110080634-edea822f558a
 	github.com/mattermost/mattermost/server/public v0.1.7-0.20240806035841-540febd866aa
 	github.com/mattermost/mattermost/server/v8 v8.0.0-20240726090344-5547504c1d68
+	github.com/opensearch-project/opensearch-go/v4 v4.1.0
```

Comment:
I see this one was added, but the `github.com/elastic/go-elasticsearch/v7` dependency was not removed. Is it still needed?

Reply:
Yeah `go mod tidy` didn't remove it. We still use some elasticsearch types for the requests I believe.